lv_objx/lv_page.c

```diff
@@ -488,13 +497,29 @@ void lv_page_focus(lv_obj_t * page, const lv_obj_t * obj, uint16_t anim_time)
         scrlable_y = -(obj_y + style_scrl->body.padding.ver + style->body.padding.ver);
         scrlable_y -= style_scrl->body.padding.ver;
         scrlable_y += page_h - obj_h;
+    }
+    /*Out of the page on the left*/
+    else if((obj_w <= page_w && left_err > 0) ||
```

Comment:
@zavovi Does this handle the case where the object is out of the page on the top and left at the same time?

Reply:
Oh, so sorry, I fixed it in next commit. I tried it and it seems working properly now.